src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.FileWriterService;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.TransactionService;
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import core.basesyntax.service.impl.TransactionServiceImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.operation.BalanceOperationHandler;
+import core.basesyntax.strategy.operation.OperationHandler;
+import core.basesyntax.strategy.operation.PurchaseOperationHandler;
+import core.basesyntax.strategy.operation.ReturnOperationHandler;
+import core.basesyntax.strategy.operation.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String SOURCE_FILE_PATH
+            = "src/main/java/core/basesyntax/resource/input.csv";
+    private static final String REPORT_FILE_PATH
+            = "src/main/java/core/basesyntax/resource/report.csv";
+    private static final StorageDao storageDao = new StorageDaoImpl();
+    private static final FileReaderService fileReaderService = new FileReaderServiceImpl();
+    private static final FileWriterService fileWriterService = new FileWriterServiceImpl();
+    private static final TransactionService transactionService = new TransactionServiceImpl();
+
+    public static void main(String[] args) {
+        Map<FruitTransaction.Operation, OperationHandler> fruitServiceMap = new HashMap<>();
+        fruitServiceMap.put(FruitTransaction.Operation.BALANCE, new BalanceOperationHandler());

```

Comment:
I think you should to extract map initialization from main method to improve readability